### PR TITLE
Display total fee rewards in USD in My Pools/Remove Liquidity pages

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
| Before | After |
| --- | --- |
| image |  image |
| image |  image |

### CODE REVIEW

File: src/components/ProAmm/ProAmmFee.tsx [hunk 1]
```diff
@@ -26,6 +26,18 @@ import { useUserSlippageTolerance } from 'state/user/hooks'
...
 
+/**
+  800                   -> '800'
+  0.123123123           -> '0.12312312'
+  0.123                 -> '0.123'
+  0.12342224234234324   -> '0.12342224'
+
+  This function is to remove trailing 0's after using toFixed
+ */
+const formatUSDValue = (val: number) => {
```
Comment (nguyenhoaidanh) [inline src/components/ProAmm/ProAmmFee.tsx:37]: we already have `formatNumberWithPrecisionRange`

File: src/pages/ProAmmPool/PositionListItem.tsx [hunk 2]
```diff
@@ -286,6 +286,10 @@ function PositionListItem({
     )
   }, 0)
 
+  const totalFeeRewardUSD =
```
Comment (nguyenhoaidanh) [inline src/pages/ProAmmPool/PositionListItem.tsx:289]: do we need big number calculation: Fraction, ... ?

File: src/components/ProAmm/ProAmmFee.tsx (hunk 1 above)
Comment (hungdoansy) [reply src/components/ProAmm/ProAmmFee.tsx]: Nice. This is exactly what I wanted

File: src/pages/ProAmmPool/PositionListItem.tsx (hunk 2 above)
Comment (hungdoansy) [reply src/pages/ProAmmPool/PositionListItem.tsx:289]: Not quite. We use this style of calculation in a few more places

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**